### ISSUE TITLE
Resources host is down

### ISSUE THREAD

**Issue by lyinch** [matthiaskoenig/brendapy#39]
Hello,

The host where the zip file is stored is down

```
$ ping 141.20.66.64 -c 3
PING 141.20.66.64 (141.20.66.64): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1

--- 141.20.66.64 ping statistics ---
3 packets transmitted, 0 packets received, 100.0% packet loss
```

This means that I can't use the package as it tries to download it on startup.

It would be ideal to add the original sources in the readme so that users can download them manually in case the aggregated zip file is unavailable again in the future. I assume that these are the sources although they are not all in json:

BRENDA: https://www.brenda-enzymes.org/download_brenda_without_registration.php
TAXONOMY_ZIP: https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/
BTO: https://www.ebi.ac.uk/ols/ontologies/bto
CHEBI: https://www.ebi.ac.uk/chebi/

Thank you

**Comment by matthiaskoenig** [comment]
Sorry for that. We currently have issus with our university network. I will have a look during the day.

**Comment by matthiaskoenig** [comment]
Sorry for the delay, things are back to normal.

**Comment by what-is-what** [comment]
We cannot have an access to 141.20.66.64.

--2022-05-13 17:20:42--  http://141.20.66.64/brendapy/brendapy-data-v0.4.0.zip
Connecting to 141.20.66.64:80... failed: Connection timed out.

Is there any solution?

**Comment by matthiaskoenig** [comment]
Will provide a solution during the day. Thanks for the report.

**Comment by matthiaskoenig** [comment]
Will be solved as part of #38, i.e. we have to update the BRENDA files and provide a stable download.
